### PR TITLE
Fix error logging in Poller for service scheduling

### PR DESCRIPTION
**Pull request by JSimo**
The error itself had accidentally been removed in the logging. 

Seeing an error in this place when an node is added and removed with the same IP address, which we want to investigate further. Sadly the error message we need is not logged.

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/${JIRA-ISSUE-NUMBER}

### CODE REVIEW

File: opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java [hunk 1]
```diff
@@ -463,7 +463,7 @@ protected void doInTransactionWithoutResult(TransactionStatus arg0) {
             node.withTreeLock(r);
 
         } catch (final Throwable e) {
-            LOG.error("Unable to schedule service {}/{}/{}", nodeId, normalizedAddress, svcName);
+            LOG.error("Unable to schedule service {}/{}/{}", nodeId, normalizedAddress, svcName, e);
```
Comment (indigo423) [inline opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:466]:
```suggestion
            LOG.error("Unable to schedule service {}/{}/{}. Exception message: {}", nodeId, normalizedAddress, svcName, e.getMessage());
```

**Review by indigo423 — COMMENTED** [review]
Suggestion to get the exception message.

File: opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java (hunk 1 above)
Comment (indigo423) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:466]: We need to look here if the Exception toString() or the getMessage() method gives useful information.

Comment (cgorantla) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:466]: Convention in our code is to log whole call stack, so the change looks fine.

Comment (cgorantla) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:466]: I would favor the original one. No need to explicitly log exception message.

Comment (JSimo) [reply opennms-services/src/main/java/org/opennms/netmgt/poller/Poller.java:466]: Sounds good to me. Signed the CLA.
Not sure if a tiny change like this requires an JIRA.

**Review by cgorantla — APPROVED** [review]
LGTM!

**Pull request merged**